deployment/v3/utils/purge-identity-cache/delete.sh

```
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+set -e
+
+# ====== CONFIGURATION ======
+NAMESPACE="clear-identity-caches"
+
+# ====== CONFIRMATION ======
+echo "WARNING: This will delete the entire namespace '$NAMESPACE' and all resources inside it!"
+read -p "Are you sure you want to continue? [y/N]: " CONFIRM
```

Shellcheck (0.11.0)

[info] read without -r will mangle backslashes.

(SC2162)